conductor-client/src/main/java/com/openlattice/postgres/ResultSetAdapters.java

```diff
@@ -75,6 +75,7 @@
 import com.openlattice.organization.ExternalTable;
 import com.openlattice.organization.OrganizationEntitySetFlag;
 import com.openlattice.organization.roles.Role;
+import com.openlattice.organizations.JdbcConnectionParameters;
```

Comment:
It looks like the only changes to this file are imports. If they are not used anywhere, please remove.